src/pzpr/variety.js

```diff
@@ -251,6 +251,7 @@
 			mochikoro: [1, 0, "モチコロ", "Mochikoro", "nurikabe"],
 			mochinyoro: [1, 0, "モチにょろ", "Mochinyoro", "nurikabe"],
 			moonsun: [0, 0, "月か太陽", "Moon or Sun", "country"],
+			myopia: [0, 0, "Myopia", "Myopia", "myopia"],
```

Comment:
```suggestion
			myopia: [0, 0, "Myopia", "Myopia"],
```

`myopia` is a base type, so the parent type doesn't need to be listed here